albumentations/core/composition.py

```diff
@@ -269,6 +269,61 @@ def __call__(self, force_apply=False, **data):
         return data
 
 
+class SomeOf(BaseCompose):
+    """Select some of the transforms to apply. Selected transforms will be called with `force_apply=True`.
+    Transforms probabilities will be normalized to one 1, so in this case transforms probabilities works as weights.
+
+    Args:
+        sample_range (int, int): Range of possible sample sizes.
+        transforms (list): list of transformations to compose.
```

Comment:
Perhaps, it make sense to make `transforms` the first argument, to follow signature of existing transforms.

Reply:
Fixed in latest commit